disnake/ext/commands/params.py

```diff
@@ -683,7 +683,16 @@ def expand_params(command: AnySlashCommand) -> List[Option]:
 
     Returns the created options
     """
-    params = collect_nested_params(command.callback)
+    sig = signature(command.callback)
+    _, inter_param, params, injections = collect_params(command.callback)
+
+    if inter_param is None:
+        raise TypeError(f"Couldn't find an interaction paramter in {command.callback}")
```

Comment:
```suggestion
        raise TypeError(f"Couldn't find an interaction parameter in {command.callback}")
```
Everything else looks fine.